app/models/house.rb

```diff
@@ -3,10 +3,27 @@ class House < ApplicationRecord
   has_many :home_owners_associations, through: :hood
   has_one :house_metadatum, dependent: :destroy
 
+  delegate :zpid, to: :house_metadatum, allow_nil: true
```

Comment:
There's no real reason to have to safe navigate into this value every time we need to access it.